hostsfile/src/lib.rs

```diff
@@ -106,11 +114,19 @@ impl HostsBuilder {
     }
 
     /// Inserts a new section to the system's default hosts file.  If there is a section with the
-    /// same tag name already, it will be replaced with the new list instead. Only Unix systems are
-    /// supported now.
+    /// same tag name already, it will be replaced with the new list instead.
     pub fn write(&self) -> Result<()> {
         let hosts_file = if cfg!(unix) {
             PathBuf::from("/etc/hosts")
+        } else if cfg!(windows) {
+            PathBuf::from(
+                // according to https://support.microsoft.com/en-us/topic/how-to-reset-the-hosts-file-back-to-the-default-c2a43f9d-e176-c6f3-e4ef-3500277a6dae
+                // the location depends on the environment variable %WinDir%.
+                format!(
+                    "{}\\System32\\Drivers\\Etc\\hosts",
+                    std::env::var("WinDir").expect("missing environment variable %WinDir%")
```

Comment:
Let's return an `std::io::ErrorKind::NotFound` type error here instead of panicking.

Reply:
Would returning the `std::env::VarError` also be okay? That could be acomplished by just using the question mark operator and simpler than instantiating a different error.

Reply:
Ah, I forgot that this crate returns a generic `Box<dyn std::error::Error>`. Yes go ahead and use a `std::env::VarError`, and we can save the error refactoring for a later time 🙂.

Reply:
👍 has been implemented that way